src/tools/rest-backend/model_pluginformat.hpp

```diff
@@ -97,10 +99,21 @@ class PluginFormat
 		return m_pluginstatuses;
 	}
 
+	/**
+	 * @brief getter for the plugin configuration
+	 * 
+	 * @returen A keyset containing the plugin config
+	 */
+	kdb::KeySet getConfig () const
```

Comment:
Return a const object? (Otherwise values of the keyset could be changed affecting m_config)